js/modules/timeline.src.js

```diff
@@ -41,11 +41,28 @@ wrap(H.SVGElement.prototype, 'animate', function (proceed, params) {
     }
     return proceed.apply(this, Array.prototype.slice.call(arguments, 1));
 });
```

Comment:
What is the reason of extending SVGElement with function? IMHO will be better to use internal function.

Reply:
I agree. Now it extends only the `dataLabel` elements.